thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js

```diff
@@ -69,6 +69,26 @@ const fetchCombinedAnomalies = (anomalyIds) => {
   }
 };
 
+/**
+ * Fetches severity scores for all anomalies
+ * TODO: Move this and other shared requests to a common service
+ * @param {Array} anomalyIds - list of all found anomaly ids
+ * @returns {RSVP promise}
+ */
+const fetchSeverityScores = (anomalyIds) => {
+  if (anomalyIds.length) {
```

Comment:
Do we need a check against anomalyIds being `undefined`/`null` here?

Reply:
yes. thanks.